targets/targets.json

```diff
@@ -3557,7 +3557,9 @@
             "SPI",
             "SPI_ASYNCH",
             "STCLK_OFF_DURING_SLEEP",
-            "TRNG"
+            "TRNG",
+            "USTICKER",
+            "LPTICKER"
```

Comment:
Please keep the list alphabetically sorted.

Reply:
fixed